EIPS/eip-5773.md

```diff
@@ -0,0 +1,503 @@
+---
+eip: 5773
+title: Multi-Resource context-dependent tokens
```

Comment:
Hm, maybe:

```suggestion
title: Multimedia Metadata for Tokens
```

Not a hard suggestion because naming is hard...

Reply:
We agreed on a new name: `Context-Dependent Multi-Asset Tokens`

While *asset* is still very broad, we added an entry to the `Rationale` explaining the need for such a broad term.